### PR TITLE
Update fivem_license max input rule to support new key format

### PR DESCRIPTION
**Pull request by LittleDraily**
fix an error with fivem licensekeys that are larger then 32 characters as fivem licensekey length grows

### CODE REVIEW

**Comment by parkervcp** [comment]
The new keys are a specific length. The should be no reason to go way past that.

Can we get the exact length please.

**Comment by Software-Noob** [comment]
> The new keys are a specific length. The should be no reason to go way past that.
> 
> Can we get the exact length please.

33

File: game_eggs/gta/fivem/egg-five-m.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
             "default_value": "",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|string|max:32"
+            "rules": "required|string|max:64"
```
Comment (Software-Noob) [inline game_eggs/gta/fivem/egg-five-m.json:38]:
```suggestion
            "rules": "required|string|max:33"
```

**Review by Software-Noob — CHANGES_REQUESTED** [review]

**Comment by Software-Noob** [comment]
I've fixed and tested it on your behalf. 

@parkervcp  I don't know about your current situation with the Fivem team, and as such will leave this up to you

**Comment by parkervcp** [comment]
All this does it fix a var length and doesn't update anything else. Just makes it easier to use the new key length.

**Review by Software-Noob — APPROVED** [review]

**Pull request merged**